website/docs/r/service_v1.html.markdown

```diff
@@ -119,6 +117,40 @@ resource "fastly_service_v1" "demo" {
 }
 ```
 
+Basic usage with [custom Director](https://docs.fastly.com/api/config#director):
```

Comment:
Thanks for your thorough docs. I may not hold onto this section specifically long term. I think we can document each of the API resources without providing full examples for each of them.